assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],

```

Comment:
Nah I think it might be a bug, or something about that workspace that can leave the setup oddly fragile. We need to look at that from a mentor/engine perspective. Maybe @xrtariq2594 has some clues but he's likely unavailable for a bit